src/modules/fw_autotune_attitude_control/fw_autotune_attitude_control.cpp

```diff
@@ -416,16 +452,12 @@ void FwAutotuneAttitudeControl::updateStateMachine(hrt_abstime now)
 		break;
 	}
 
-	// In case of convergence timeout or pilot intervention,
+	// In case of convergence timeout
 	// the identification sequence is aborted immediately
-	manual_control_setpoint_s manual_control_setpoint{};
-	_manual_control_setpoint_sub.copy(&manual_control_setpoint);
-
 	if (_state != state::wait_for_disarm
 	    && _state != state::idle
-	    && (((now - _state_start_time) > 20_s)
-		|| (fabsf(manual_control_setpoint.x) > 0.2f)
-		|| (fabsf(manual_control_setpoint.y) > 0.2f))) {
+	    && !rc_switch_enabled
+	    && ((now - _state_start_time) > 20_s)) {
```

Comment:
I think that if the user doesn't have an RC switch mapped to start and stop auto-tuning, we should still keep the possibility to abort using the sticks

Reply:
What if we aborted on a mode change regardless of if RC aux enable switch is set? I really think it's important for manual control to be possible at all times. I'm open to discussing it, of course. I think anytime we take manual control away from the user it leads to an uncomfortable experience.

Reply:
That sounds like a really good idea actually: aborting on mode change